<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>797</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>797</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>50</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="profile_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Book Antiqua</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="dl_frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>631</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="loadButton_dl">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load Model</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createButton_dl">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create Model</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ml_or_dl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>631</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="machine_learning">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Machine Learning</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deep_learning">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Deep Learning</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="main_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QFrame" name="ml_frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>631</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="loadButton_ml">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Load Model</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createButton_ml">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create Model</string>
     </property>
    </widget>
   </widget>
   <zorder>ml_frame</zorder>
   <zorder>ml_or_dl</zorder>
   <zorder>logout</zorder>
   <zorder>frame</zorder>
   <zorder>dl_frame</zorder>
   <zorder>main_menu</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
